app/models/post.rb

```diff
@@ -3,6 +3,10 @@ class Post < ApplicationRecord
   has_many :comments
   has_many :likes
 
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, numericality: { greater_than_or_equal_to: 0 }
+  validates :likes_counter, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
- [ ] I think it would be great if you add validation to `Only integre`  since it is mentioned in the requirements so I recommend adding it you can check this link so you can add like this [ruby validation](https://guides.rubyonrails.org/active_record_validations.html) ✅💯


![image](https://github.com/Lawmsangi/blog-app/assets/58553711/44e66dbf-5566-4e91-9cfd-fd82fa802355)